lerobot/configs/default.yaml

```diff
@@ -120,7 +120,7 @@ eval:
   # `batch_size` specifies the number of environments to use in a gym.vector.VectorEnv.
   batch_size: 1
   # `use_async_envs` specifies whether to use asynchronous environments (multiprocessing).
-  use_async_envs: false
+  use_async_envs: true
```

Comment:
This doesn't work with the ACT environment, I think due to some EGL stuff. Which is why I defaulted it to `false`.

Reply:
OK, I think we can let it to be `true` by default, by set it to `false` in the ACT env

Reply:
Okay. You'll also have to set it to False in the Xarm env please.

Reply:
I just added it!